### DISCUSSION TITLE
Why doesn't renovate honor pr and branch limit per repository on selfhosted renovate

### DISCUSSION THREAD

**Discussion by janorn** [renovatebot/renovate#22315] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

"35.96.1"

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I want to be able to get more than 10 PRs from specific repositories. This worked up until 4 months ago when the new limits was introduced. 

### What you tried so far.

I added this config to the repo specific config.

```json5
  // Laters version of renovate impose a limit of 10 PRs/Branches 
  "prConcurrentLimit": 0,
  "branchConcurrentLimit": 0,
```

### Relevant debug logs

From the debug log it looks like renovate picks it up. However I get a lot of results like this:
<details><summary>Logs</summary>
 
 ```json
{
    "branchName": "renovate/datadog.datadog-4.x",
    "prNo": null,
    "prTitle": "Update dependency datadog.datadog to v4.19.0 [ci skip]",
    "result": "branch-limit-reached",
    "upgrades": [
        {
            "datasource": "git-tags",
            "depName": "datadog.datadog",
            "displayPending": "",
            "fixedVersion": "4.18.0",
            "currentVersion": "4.18.0",
            "currentValue": "4.18.0",
            "newValue": "4.19.0",
            "newVersion": "4.19.0",
            "packageFile": "ansible/imported_roles/requirements.yml",
            "updateType": "minor",
            "packageName": "https://github.com/DataDog/ansible-datadog/"
        }
    ]
}
 ```

</details>

**Comment by rarkins** [comment, marked as answer]
Perhaps it's `prHourlyLimit` instead?

**Reply by janorn** [reply]
You are probably right!

**Reply by janorn** [reply]
I bumped hourly to 10. And now I see PRs coming in. Will observe it a bit more before I mark it as answered.

Still it would be much better if renovate could log what limit it hit instead of same error message all the time...

**Reply by rarkins** [reply]
PRs welcome
